packages/help-center/src/happychat-window-communicator.ts

```diff
@@ -59,5 +82,5 @@ export function useHCWindowCommunicator(
 		return () => {
 			window.removeEventListener( 'message', messageHandler, false );
```

Comment:
The `false` param is irrelevant here, as `message` is not a DOM event and doesn't traverse the DOM tree.